alternator/consumed_capacity.cc

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2024-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#include "consumed_capacity.hh"
+#include <seastar/core/metrics.hh>
+#include "log.hh"
+
+namespace alternator {
+
+static uint64_t MULTIPLIER = 2ULL;
+static uint64_t KB = 1024ULL;
+consumed_capacity::consumed_capacity() {
+    _metrics.add_group("alternator", {
+                   seastar::metrics::make_counter("rcu_total", seastar::metrics::description("the total RCU consumed"), _rcu).set_skip_when_empty(),
+                   seastar::metrics::make_counter("wcu_total", seastar::metrics::description("the total WCU consumed"), _wcu).set_skip_when_empty()
+    });
+}
+
+uint64_t add_consumption_helper::get_units(uint64_t length) const noexcept {
+    uint64_t bytes = MULTIPLIER*((_units + length -1)/length);
+
+    if (_isTransaction) {
+        bytes *= 2;
+    }
+    if (_isQuorum) {
```

Comment:
Does it matter if the quorum is of 2/3, or 4/5 (meaning, it's related also to the RF) ?